src/panels/config/voice-assistants/dialog-voice-assistant-pipeline-detail.ts

```diff
@@ -279,6 +286,19 @@ export class DialogVoiceAssistantPipelineDetail extends LitElement {
     }
   }
 
+  private _supportedConversationLanguagesChanged(ev) {
```

Comment:
```suggestion
  private _supportedConversationLanguagesChanged(ev: CustomEvent<{ value: "*" | string[] | undefined ) {
```
The value can be "*" according to the event type, so the `.includes` below may cause an error